build/README.md

```diff
@@ -6,12 +6,46 @@ The latest build can be downloaded from: https://quay.io/odsci/ods-ci:latest
 eg: podman pull quay.io/odsci/ods-ci:latest
 
 ```bash
+## I assume you have yq.
+## get oc from your own cluster
+
+oc_url="$(yq  e '.OCP_CONSOLE_URL' ./test-variables.yml \
+    | sed 's/console\-/downloads\-/g' )amd64/linux/oc.tar" ; echo $oc_url
+
+curl -L ${oc_url} \
+  -o - | tar xf - > ./oc
+
 # Build the container (optional if you dont want to use the latest from quay.io/odsci)
-$ podman build -t ods-ci:master -f build/Dockerfile .
+podman build -t ods-ci:master -f build/Dockerfile .
+podman build -t ods-ci:v2 -f build/Dockerfile .
+podman build -t ods-ci:v3 -f build/Dockerfile .
+
+# create the output directory
+mkdir -p $PWD/test-output
+
+# Run a single test
+# use the image as-is.
+podman run --rm -it \
+    -v $PWD/test-variables.yml:/tmp/ods-ci/test-variables.yml:Z \
+    -v $PWD/test-output:/tmp/ods-ci/test-output:Z \
+    -e RUN_SCRIPT_ARGS='--test-case tests/Tests/500__jupyterhub/test-jupyterlab-git-notebook.robot'  \
+    ods-ci:v3
+
+# a new test I'm working on:
+podman run --rm -it \
+    -v $PWD/test-variables.yml:/tmp/ods-ci/test-variables.yml:Z \
+    -v $PWD/test-output:/tmp/ods-ci/test-output:Z \
+    -e RUN_SCRIPT_ARGS='--test-case tests/Tests/500__jupyterhub/test-jupyterlab-cpu-stresstest.robot'  \
+    ods-ci:v3
+
+# if you want to run a new test without having re-built the image, just mount it:
+podman run --rm -it \
+    -v $PWD/test-variables.yml:/tmp/ods-ci/test-variables.yml:Z \
+    -v $PWD/test-output:/tmp/ods-ci/test-output:Z \
+    -v $PWD/tests/Tests/500__jupyterhub/test-jupyterlab-cpu-stresstest.robot:/tmp/ods-ci/tests/Tests/700/test.robot:Z \
+    -e RUN_SCRIPT_ARGS='--test-case tests/Tests/700/test.robot'  \
```

Comment:
When I tried running with specific tag, with the below env variable, getting errors since we have special chars and spaces.
--env RUN_SCRIPT_ARGS="--extra-robot-args '-i ODS-935'" 
Are you able to run this specific test by passing this env variable?

Reply:
I am not quite sure how tags work in RobotFramework, and I don't know what the `-i` option is doing. 

All i get when I try it is: 
```
Unknown command line switch: ODS-935'
```

the new test (`test-jupyterlab-cpu-stresstest.robot`) is just my first attempt at creating my own test, it's not important at all.  important. 

And the commands in the readme are just there as examples of the syntax.

Reply:
the same error I am also getting, we could open a new bug for this specific issue.

Reply:
@erwangranger I've integrated some of your changes in PR #205 along with some changes to make the test selection flag working. 
Thanks!